README.md

```diff
@@ -1,2 +1,42 @@
-# java-http-client-template
-Template for quickly creating a client in Java
+# { Client }
+{ Client description }
+
+## Install
+
+### 1.Add maven dependency
+
+```
+<dependency>
+    <groupId>com.byndyusoft.restclient</groupId>
+    <artifactId>{Client}</artifactId>
+    <version>{Veriosn}</version>
+</dependency>
+```
+
+### 2. Add configuration properties
+```
+clients:
+  {client}:
+    readTimeout: 10s            # required
+    connectionTimeout: 10s      # required
+    connectionManager:
+      defaultMaxPerRoute: 50    # required
+      maxTotal: 50              # required
+    baseUrl: localhost:8080     # required
```

Comment:
Why did you put the baseUrl at the very end? It's not intuitive, imho

Reply:
done